zeppelin-server/derby.log

```diff
@@ -0,0 +1,13 @@
+----------------------------------------------------------------
```

Comment:
It looks like both were an issue.  I accidentally added it, and it wasn't in the gitignore.  When I deleted the file and removed it from git, after re-running `mvn clean package`, git wanted to add the file again.  I've added a line to the .gitingore file to prevent this from happening again.